distributor-api-v1/general-guidelines.md

```diff
@@ -19,6 +19,23 @@ The API accepts only`HTTP POST`requests with`Content-Type`set to`application/jso
 * **Operation**
   * Name of the operation to be performed.
 
+### Body <a id="body"></a>
```

Comment:
I know, that those weird links were already in distributor doc, but they are useless because of markdown targeting to headline titles by default. I would remove them, but because we will write new API, make a note, that will not use it in the new DOC.